runtime/lua/vim/lsp/util.lua

```diff
@@ -1318,6 +1318,9 @@ function M.make_text_document_params()
   return { uri = vim.uri_from_bufnr(0) }
 end
 
+function M.make_workspace_params()
```

Comment:
This could take `added` and `removed` as params, and then if they are nil, do empty tables.

Reply:
I took a pass at this, do you think I should pass named arguments instead for clarity and do the nil check? See: 8d186fa0f89b4039f7eeeae6fe0316c3bef9dd7b